src/reducers/workspaces.ts

```diff
@@ -4,6 +4,7 @@ import {
   CHANGE_ACTIVE_TAB,
   CHANGE_CHAPTER,
   CHANGE_EDITOR_WIDTH,
+  CHANGE_LIBRARY,
```

Comment:
CHANGE_EXTERNALS, since the prop name to workspace is externals. Mirrors CHANGE_CHAPTER for prop name of chapter.